src/components/Heading/Heading.tsx

```diff
@@ -3,14 +3,15 @@ import React, { forwardRef } from 'react';
 import styles from './Heading.module.css';
 
 export const VARIANTS = [
-  'error',
-  'base',
+  'neutralSubtle',
+  'neutralMedium',
+  'neutralStrong',
```

Comment:
I also reorganized the variants here and in the stories so they feel more organized.

Reply:
**blocking**: is it easier to keep these names kebab cased (e.g. `neutral-subtle`)? that's consistent with the multi-word size values and would mean we can get rid of the extra `variantComputed` code

Reply:
Hmm I feel like we usually use camel case for props, like `className`, `onDismiss`, and `descriptionAs`. I feel like the sizes should be changed because I think that's the only place we use kebab in props.

Reply:
I can't find any multi-word string prop values beyond the text sizes and kebab class names.

Reply:
No wait I found some more:
- `position` for `ButtonDropdown`: 'top-left' | 'top-right' | 'bottom-right'
- `variant` for `Grid`: '2up' | 'side-by-side' | '1-3up' | '3up' | '4up' | '1-2-4up' | '1-2-1up'

That's a definite pattern. Will update. If this isn't in the documentation yet we should probably add it.

Reply:
ooh nice digging, I didn't realize we had those. ya +1 adding to documentation 👍